app/payment/session_test.go

```diff
@@ -161,16 +161,16 @@ func Test_SubPayChProposals(t *testing.T) {
 		})).Return(nil)
 
 		gotErr := payment.SubPayChProposals(sessionAPI, dummyNotifier)
-		require.NoError(t, gotErr)
-		require.NotNil(t, notifier)
+		assert.NoError(t, gotErr)
+		assert.Nil(t, gotErr)
```

Comment:
What is the reason for changing `require` to `assert` and removing `NotNil` check on notifier  ?

Reply:
Keeping the older format.